packages/components/List/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import SimpleCard from '../SimpleCard';
+import SearchBar from '../SearchBar';
+
+const List = ({ items, handleSearch, handleEdit, handleDelete }) => {
+
+    const includesText = (textOne, textTwo) => {
+        return textOne.toLowerCase().trim().includes(textTwo.toLowerCase().trim());
+    };
+
+    const [list, handleUpdateList] = useState(items);
+
+    handleSearch = searchText => {
+        const results = items.filter(item => {
+            if (
+                includesText(item.title, searchText) ||
+                includesText(item.description, searchText)
+            ) {
+                item.display = true;
+
+                return item;
+            } else {
+                item.display = false;
+
+                return item;
+            }
+        });
+        handleUpdateList(results);
+    };
+
+      let searchEl;
+      if (items !== null && items.length !== 0) {
+          searchEl = <SearchBar
+              handleSearch={handleSearch}
+          />;
```

Comment:
you can do the:
```const listItems = list.map((item)...```
here.

you also don't need to do `if (item.display) ...` - because you have updated `items` and they should be the new Items?

Reply:
`Display: true` is being used to return cards that match search input. I think we do need this so when the list component re-renders, it only renders the items that matched the last entered search input.